tensordict/tensordict.py

```diff
@@ -1477,6 +1477,84 @@ def apply(
             out.lock_()
         return out
 
+    def map(
+        self,
+        fn: Callable,
+        dim: int = 0,
+        num_workers: int = None,
+        chunksize: int = None,
+        pool: mp.Pool = None,
+    ):
+        """Maps a function to splits of the tensordict across one dimension.
+
+        This method will apply a function to a tensordict instance by chunking
+        it in tensordicts of equal size and dispatching the operations over the
+        desired number of workers.
+
+        The function signature should be ``Callabe[[TensorDict], Union[TensorDict, Tensor]]``.
+        The output must support the :func:`torch.cat` operation. The function
+        must be serializable.
+
+        Args:
+            fn (callable): function to apply to the tensordict.
+                Signatures similar to ``Callabe[[TensorDict], Union[TensorDict, Tensor]]``
+                are supported.
+            dim (int, optional): the dim along which the tensordict will be chunked.
+            num_workers (int, optional): the number of workers. Exclusive with ``pool``.
+                If none is provided, the number of workers will be set to the
+                number of cpus available.
+            chunksize (int, optional): the number of chunks to split the tensordict
```

Comment:
if it is the number of chucks it is not the chunksize right? isn't it the num_chuncks?

Reply:
yeah we can we all the options. 
the thing i was saying is that when something refers to the size of the chunks it should be called smth that contains "size" and when something refers to the number of chunks it should contain "num"